spec/features/users_can_switch_organisation_spec.rb

```diff
@@ -101,6 +101,47 @@
     end
   end
 
+  context "when a BEIS user has switched organisation" do
+    let(:user_with_additional_organisations) do
+      org1 = create(:partner_organisation)
+      org2 = create(:partner_organisation)
+      org3 = create(:beis_organisation)
+      user = create(:beis_user)
+      user.additional_organisations << [org1, org2, org3]
+      user
```

Comment:
```suggestion
      user = create(:beis_user)
      user.additional_organisations << [
        create(:partner_organisation), 
        create(:partner_organisation), 
        create(:beis_organisation)
      ]
      user
```

This is about the only thing I'd be tempted to do, just so my brain doesn't try and equate the `org1` and `org2` up here with the `org1` and `org2` down there